app/models/miq_queue.rb

```diff
@@ -690,7 +679,7 @@ def self.messaging_protocol
   private_class_method def self.messaging_options_from_file
     return unless MESSAGING_CONFIG_FILE.file?
 
-    YAML.load_file(MESSAGING_CONFIG_FILE)[Rails.env].symbolize_keys.tap { |h| h[:host] = h.delete(:hostname) }
```

Comment:
Not sure why we were storing `:hostname` then switching it to `:host` later instead of just setting it properly in the messaging.yml file

Reply:
This might have been for compatibility with / matching database.yml so the look and feel was consistent.  It also could have been that Artemis took "hostname", but probably the former.

Reply:
The artemis/stomp client also takes `:host` as an option https://github.com/ManageIQ/manageiq-messaging/blob/master/lib/manageiq/messaging/stomp/client.rb#L64 so I'm guessing it was the former as well.

Reply:
and Rails itself takes `:host` now that I actually checked, so I have no idea why this was `:hostname`...I'm +1 for moving to `:host`